react-native-libraries.json

```diff
@@ -6171,6 +6171,17 @@
     "macos": false,
     "unmaintained": false
   },
+  {
+    "githubUrl": "https://github.com/microsoft/fluentui-react-native",
+    "npmPkg": "@fluentui/react-native",
+    "ios": true,
+    "android": false,
+    "web": false,
+    "expo": true,
+    "windows": true,
+    "macos": false,
+    "unmaintained": false
```

Comment:
As one of the maintainers of fluentui-react-native, I'm not sure what some of these fields mean, but naively I would expect macos to be true as well as android and maybe expo should be false?

Reply:
@FalseLobster Thank you for the response! 👍 

The description of the fields can be found in Readme - https://github.com/react-native-community/directory#library-fields-description - basically they mean if the package works with selected out of three platform (links are in Readme). 🙂 

Also this PR is quite old and needs rebase as well as the package should be moved at the end of the list, @luism3861 would you like to take care of that?

Reply:
@FalseLobster - does fluentui-react-native ship native code or does it just depend on react-native primitives? if it just depends on react-native primitives (or common libraries used in the ecosystem, like react-native-gesture-handler, reanimated, and [others included in Expo SDK](https://docs.expo.dev/versions/latest/)) then it will work fine in expo go and on https://snack.expo.io